Sources/Provider/ItemProvider.swift

```diff
@@ -246,6 +247,22 @@ extension ItemProvider: Provider {
                 .eraseToAnyPublisher()
     }
     
+    public func asyncProvide<Item>(request: any ProviderRequest, decoder: ItemDecoder = JSONDecoder(), providerBehaviors: [ProviderBehavior] = [], requestBehaviors: [RequestBehavior] = []) async -> Result<Item, ProviderError> where Item : Identifiable, Item : Decodable, Item : Encodable {
```

Comment:
Can `Item` be simply `Providable` in the declaration instead of all the conformances at the end, to match the other declarations? e.g. `asyncProvide<Item: Providable>`